### ISSUE TITLE
[CoE Starter Kit - BUG] Update to core 4.4.2 Admin | Sync Template v3 (Environment Properties) issue

### ISSUE THREAD

**Issue by Fryimb** [microsoft/coe-starter-kit#5393]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Hello,

Just updated to April Updatewith core 4.4.2 from 3.56
But When i want to activate the flow Admin | Sync Template v3 (Environment Properties) i have this error "Flow client error returned with status code "BadRequest" and details "{"error":{"code":"ChildFlowNeverPublished","message":"The workflow with id '5c248f24-f7c9-ed11-b597-0022480813ff' cannot be used as a child workflow because it has never been published. Child workflows need to be published at least once before they can be included in a published parent workflow."}}"."

I havent find this child flow in the core solution.

There also a message in run history " problem with flow trigger" and when i click " resolve issue" and in error detail "it seems that the operation of your flow has reached an action limit designed to protect the connector service called" or "Combined execution time of incoming requests exceeded limit of 1200000 milliseconds over time window of 300 seconds. Decrease number of concurrent requests or reduce the duration of requests and try again later."

### Expected Behavior

[
![coe core 4 4 2 update flow error](https://user-images.githubusercontent.com/105657326/234796888-bac3f718-b7de-4595-9da2-65a0741f95ac.PNG)
](url)
![flow number error](https://user-images.githubusercontent.com/105657326/234797459-41a73f7c-d599-4d5d-91d6-625d68f080cf.PNG)
![coe flow env error](https://user-images.githubusercontent.com/105657326/234797706-1755f73e-28bc-481b-a517-b55b8a4d4a38.PNG)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.4.2

### What app or flow are you having the issue with?

Admin | Sync Template v3 (Environment Properties)

### What method are you using to get inventory and telemetry?

Cloud flows

### Steps To Reproduce

1. In COE environment remove custom layer
2. Import Core Solution with upgrade action
3. Go to core solution
4. Click active the flow Admin | Sync Template v3 (Environment Properties)

**Comment by DominicMeznik** [comment]
Hello! Make sure to turn the child flow on first: SYNC HELPER - Get Security Role Users.

This error occurs, if you´re trying to turn on the parent before the referenced child flow.

About the 2nd issue I´m not really sure what to do

**Comment by StylezDevops** [comment]
Is this actually documented anywhere? 
Seeing the same error here... 
![image](https://user-images.githubusercontent.com/94297031/234882488-a6e7e765-257a-4493-b675-807bda8486dc.png)

The flow: 'Admin | Sync Template v3 (App Properties)' - is also undocumented

Are there any other undocumented flows for those of us who need to perform deployments in an automated fashion and may not be able to run the "Set Up Wizard" ?

Any ETA on when the documentation will accurately reflect new flows, flow dependencies and so on? 
This information is critical to performing automated deployment of CoE into enteprise organisations.
thanks

**Comment by DominicMeznik** [comment]
@StylezDevops 

I´m not sure where or if it´s documented, I just check dependencies of a component if I encounter this error:

![image](https://user-images.githubusercontent.com/109345362/234884427-a7cde69f-b6fc-4d97-9fea-8dab5001465d.png)

I assume it´s really hard to keep everything up to date in the docs, while also developing the whole Center of Excellence Solution. 

Please do not forget that there are only 2 people doing everything.

**Comment by Jenefer-Monroe** [comment]
Thanks everyone. Also remember that if you use the Setup Wizard to install or upgrade, then the correct flows and their setup order is baked in to setting up for you!

**Comment by StylezDevops** [comment]
Yes Thanks - I do understand it can be difficult to document. And yes, I am moving past the errors as I encounter them.
The Setup wizard, whilst a fantastic piece of work, is not an option for this project. 

You guys do fantastic work and it must be incredibly difficult to stay on top of documentation. I understand this.

I am aware the flow i mentioned has already been raised as a documentation bug. 
Regards

**Comment by Kreangkraik** [comment]
**Currently (Mar/12/2024) I can't find this flow "Admin | Sync Template v3 (Environment Properties)" anymore. What happened?**

![Environment Properties is missing](https://github.com/microsoft/coe-starter-kit/assets/163093060/02f354a5-44a8-401f-af4a-ef91548a5d3f)
